projectDocs/dev/userGuideStandards.md

```diff
@@ -18,13 +18,17 @@ Feature flags, comboboxes and checkboxes should be included using the following
 `FeatureDescriptionAnchor` should not include the settings category.
 Once the anchor is set it cannot be updated, while settings may move categories.
 
-```text2tags
-==== The name of a feature ====[FeatureDescriptionAnchor]
-: Default
-  Enabled
-: Options
-  Default (Enabled), Enabled, Disabled
-:
+```md
+|||
+|-|-|
+| Options | Default (Enabled), Enabled, Disabled |
+| Default | Enabled |
+| Toggle command | `nvda+shift+e` |
+
+| Option | Behaviour |

```

Comment:
This PR should be focused on format conversion to something compatible with markDown, i.e. converting the existing definition lists to tables. Other changes that can lead to discussion and opinion have not been fully discussed (there are unanswered comments); thus I'd really encourage to handle them in a separate PR or issue.

Thus, I'd recommend to remove this new table from User Guide standards.